joss/paper.bib

```diff
@@ -96,19 +111,25 @@ @article{10.1145/321607.321609
   numpages = {14}
 }
 
-@article{Schoenberg1946ContributionsTT,
-  title={Contributions to the problem of approximation of equidistant data by analytic functions. Part A. On the problem of smoothing or graduation. A first class of analytic approximation formulae},
-  author={I. J. Schoenberg},
-  journal={Quarterly of Applied Mathematics},
-  year={1946},
-  volume={4},
-  pages={45-99},
-  url={https://api.semanticscholar.org/CorpusID:125923957}
+@Inbook{Schoenberg1988,
+  author="Schoenberg, I. J.",
+  editor="de Boor, Carl",
+  title="Contributions to the Problem of Approximation of Equidistant Data by Analytic Functions",
+  bookTitle="I. J. Schoenberg Selected Papers",
+  year="1988",
+  publisher="Birkh{\"a}user Boston",
+  address="Boston, MA",
+  pages="3--57",
+  abstract="Introduction. Let there be given a sequence of ordinates<math display='block'><mrow><mrow><mo>{\{}</mo> <mrow><msub><mi>y</mi><mi>n</mi></msub></mrow> <mo>{\}}</mo></mrow><mtext>{\&}{\#}x2003;</mtext><mrow><mo>(</mo><mrow><mi>n</mi><mo>=</mo><mn>0</mn><mo>,</mo><mo>{\&}{\#}x00B1;</mo><mn>1</mn><mo>{\&}{\#}x00B1;</mo><mn>2</mn><mo>,</mo><mo>{\&}{\#}x2026;</mo></mrow><mo>)</mo></mrow><mo>,</mo></mrow></math>{\$}{\$}{\backslash}left{\backslash}{\{} {\{}{\{}y{\_}n{\}}{\}} {\backslash}right{\backslash}{\}}{\backslash}quad {\backslash}left( {\{}n = 0, {\backslash}pm 1 {\backslash}pm 2, {\backslash}ldots {\}} {\backslash}right),{\$}{\$}corresponding to all integral values of the variable x = n. If these ordinates are the values of a known analytic function F(x), then the problem of interpolation between these ordinates has an obvious and precise meaning: we are required to compute intermediate values F(x) to the same accuracy to which the ordinates are known. Undoubtedly, the most convenient tool for the solution of this problem is the polynomial central interpolation method. It uses the polynomial of degree k --- 1, interpolating k successive ordinates, as an approximation to F(x) only within a unit interval in x, centrally located with respect to its k defining ordinates. Assuming k fixed, successive approximating arcs for F(x) are thus obtained which present discontinuities on passing from one arc to the next if k is odd, or discontinuities in their first derivatives if k is even (see section 2.121). Actually these discontinuities are irrelevant in our present case of an analytic function F(x). Indeed, if the interpolated values obtained are sufficiently accurate, these discontinuities will be apparent only if we force the computation beyond the intrinsic accuracy of the yn.",
```

Comment:
that's the worst abstract form ever 😅 who does a display block in abstract?!?!?!?!

Reply:
yeah, thats weird.... I took it from springer - https://link.springer.com/chapter/10.1007/978-1-4899-0433-1_1#citeas 😅

Reply:
I have removed it in [7dce110](https://github.com/SciML/DataInterpolations.jl/pull/265/commits/7dce1101f1edc8617cbba36b9c003a5eca6b28bf). The abstract is the first paragraph of the chapter, so not a real abstract.